assignments/can_drive/can_drive.js

```diff
@@ -1,5 +1,12 @@
 function CanDrive(hasDrivingLiscence, isTired, isSober) {
     // Write you code here
+    if(hasDrivingLiscence === false && isTired === true &&  isSober === true){
```

Comment:
what will be the output for False true false ?

Reply:
You cannot drive

Reply:
You shouldn't drive

Reply:
I made the changes again you can check it.